sig_gen_util/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "signatures"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+anyhow = "1.0.39"
+fuel-tx = "0.1"
+fuel-vm = "0.1"
+hex = "0.4"
+secp256k1 = { version = "0.20", features = ["recovery"] }
+
+[workspace]
```

Comment:
```suggestion
```

we don't need this here

Reply:
FIxed. I had to add `sig-gen-util` to the "members" subsection of the main cargo.toml.